src/mmw/apps/modeling/models.py

```diff
@@ -14,6 +14,10 @@ def project_filename(project, filename):
     return 'project_{0}/{1}'.format(project.id, filename)
```

Comment:
Unfortunately we can't delete this because it is referenced in this later-overridden migration:

https://github.com/WikiWatershed/model-my-watershed/blob/a2fe4d6e2fc035ed22cfbc5a55e91cc440248592/src/mmw/apps/modeling/migrations/0030_custom_weather_dataset.py#L19